README.md

```diff
@@ -10,6 +10,10 @@ Frontend administration application for the digital marketplace.
 
 ## Quickstart
 
+System requirements include a working node.js and npm.
+
+- note that on macOS, a `brew install` of the latest node/npm results in `Error: Cannot find module 'internal/fs'` - use nodejs.org instead
```

Comment:
I'm not sure that entirely makes sense either: once you're using nvm, version 7 is fine too. I think mentioning specific versions here can only get out-of-date, but it's useful to mention the error and some possible fixes, in case it affects other people in future.

I'll try and re-word once more.